ui/jobs/components/smn.ts

```diff
@@ -74,13 +79,13 @@ export class SMNComponent extends BaseComponent {
   }
 
   override onYouGainEffect(id: string, matches: PartialFieldMatches<'GainsEffect'>): void {
-    if (id === EffectId.FurtherRuin) {
+    if (id === '4BC') { // Further Ruin changed id in 6.0, EffectId.FurtherRuin is the 6.0 version.
```

Comment:
Could you add `FurtherRuin5x: 4BC` into ui/jobs/constants?

Reply:
Done!